css/index.css

```diff
@@ -1,3 +1,192 @@
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  /* border:1px solid red; */
+  max-width: 880px;
+  /* TB LR */
+  margin: 27px auto;
+}
+
+/* header */
+
+header {
+  /* border:1px solid green; */
+  display:flex;
+  justify-content:space-between;
+}
+
+header nav {
+  /* border:1px solid red; */
+  display:flex;
+  justify-content:space-between;
+  align-items: center;
+  width:69%
+}
+
+header nav a {
+  font-size: 16px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+
+/* cta */
+.cta {
+  /* border:1px solid red; */
+  display:flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin:68px 0;
+
+}
+
+.cta .cta-text {
+  /* border:1px solid green; */
+
+  display:flex;
+  flex-direction:column;
+  justify-content: center;
+  align-items: center;
+
+  font-size: 75px;
+  text-align: center;
+  /* margin-right:20px; */
+  align-self:center;
+  margin-left:112px;
+}
+
+.cta .cta-text button {
+  background: white;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+
+/* MAIN BODY CONTENT */
+
+.main-content {
+  /* border:1px solid red; */
+  display:flex;
+  flex-direction:column;
+  justify-content: center;
+  align-items:space-between;
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  /* border:1px solid green; */
+  display:flex;
+
+}
+
+.main-content .top-content .text-content {
+  /* border:1px solid cyan; */
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
```

Comment:
all images will also need to have a percentage-based width to be fully responsive above your breakpoints